client/src/components/auth/authComponents/LoginForm.jsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
This file is the same as previous `Login.js`, just that removed the `Don't have account yet? Sign up today!` link beside the button.